lib/widgets/fact_file/editAnswer.dart

```diff
@@ -9,7 +9,7 @@ class EditAnswer extends StatelessWidget {
       "To edit your answer, re-scan the QR code.",
       style: TextStyle(
         fontSize:
-            Screen.isTablet(context) ? 25.0 : Screen.isSmall(context) ? 14 : 20,
+            Screen.isTablet(context) ? 25.0 : Screen.isSmall(context) ? 12 : 20,
```

Comment:
This wouldn't affect my phone as it's not less than 350 wide

Reply:
surely not what is the logical pixels